privilege/privileges/tidb_auth_token_test.go

```diff
@@ -0,0 +1,410 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package privileges
+
+import (
+	"crypto/rsa"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	jwaRepo "github.com/lestrrat-go/jwx/v2/jwa"
+	jwkRepo "github.com/lestrrat-go/jwx/v2/jwk"
+	jwsRepo "github.com/lestrrat-go/jwx/v2/jws"
+	jwtRepo "github.com/lestrrat-go/jwx/v2/jwt"
+	"github.com/lestrrat-go/jwx/v2/jwt/openid"
+	"github.com/pingcap/tidb/util/hack"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	privateKeyStrings = []string{`-----BEGIN RSA PRIVATE KEY-----
```

Comment:
The certs can be generated by `util.CreateCertificates()`. Just a suggestion, it's up to you.